### PR TITLE
feat: React and i18next tutorial ARTICLES-16

### PR DESCRIPTION
**Pull request by JanCizmar**

### CODE REVIEW

**Review by stepan662 — APPROVED** [review]

File: blog/2022-01-20-super-fast-react-localizaton-with-i18next-and-tolgee.md [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+i18next [here](https://www.i18next.com/).
+
+### OK! And what is [Tolgee](https://tolgee.io)?
+
+Tolgee is open-source tool combining localization platform and integrations to provide simple way to translate the web
+applications for both developers and translators. 👨‍💻👩🏻‍💻🧖🏼👩🏻‍💻🧖🏼
+
+Tolgee Localization Platform is a place, where you can manage all your localization strings in the simple UI. It's made
+to simplify the localization process as much as possible. With Tolgee you can translate the localization strings
+directly in the Application you develop without editing localization data files. You can also automatically generate
```
Comment (marketachalupnikova) [inline blog/2022-01-20-super-fast-react-localizaton-with-i18next-and-tolgee.md:33]: application

File: blog/2022-01-20-super-fast-react-localizaton-with-i18next-and-tolgee.md [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+```
+
+Great! Now you are able to switch the language!
+
+![Language switching](/img/blog/2022-01-20-super-fast-react-localizaton-with-i18next-and-tolgee/lng-switch.gif)
+
+## Preparing for production
+
+In production mode you **don't want to leak you API key**. You want your translations to be part of the production
+build. There are multiple options to obtain you exported localization files.
```
Comment (marketachalupnikova) [inline blog/2022-01-20-super-fast-react-localizaton-with-i18next-and-tolgee.md:253]: your

File: blog/2022-01-20-super-fast-react-localizaton-with-i18next-and-tolgee.md [hunk 3]
```diff
@@ -0,0 +1,340 @@
...
+
+authors: [jcizmar]
+
+tags: [react, i18n, i18next, javascript, typescript, integration]
+---
+
+So you develop an app in React and you want many users to use it, right? To make your app usable for users in foreign
+countries, you have to translate your App to their languages. 🇬🇧 🇨🇳 🇮🇹 🇪🇸 🇫🇷
+
+In this article I am going to show you, how to integrate i18next and Tolgee to your project and how easy and fast you
```
Comment (marketachalupnikova) [inline blog/2022-01-20-super-fast-react-localizaton-with-i18next-and-tolgee.md:14]: In this article,

Comment (marketachalupnikova) [reply blog/2022-01-20-super-fast-react-localizaton-with-i18next-and-tolgee.md:14]: into your project?

File: blog/2022-01-20-super-fast-react-localizaton-with-i18next-and-tolgee.md [hunk 4]
```diff
@@ -0,0 +1,340 @@
...
+can translate a React app using these tools.
+
+<!--truncate-->
+
+### What is [i18next](https://www.i18next.com/)?
+
+`i18next` is a library enabling you to localize your app. Basically, in your code you are calling methods of this
```
Comment (marketachalupnikova) [inline blog/2022-01-20-super-fast-react-localizaton-with-i18next-and-tolgee.md:21]: in your code,

File: blog/2022-01-20-super-fast-react-localizaton-with-i18next-and-tolgee.md [hunk 5]
```diff
@@ -0,0 +1,340 @@
...
+library providing keys to be translated. The methods return value in specified language retrieved from localization
+files. There is much more about i18next, but let's keep it simple for this tutorial. You can read more about
+i18next [here](https://www.i18next.com/).
+
+### OK! And what is [Tolgee](https://tolgee.io)?
+
+Tolgee is open-source tool combining localization platform and integrations to provide simple way to translate the web
```
Comment (marketachalupnikova) [inline blog/2022-01-20-super-fast-react-localizaton-with-i18next-and-tolgee.md:28]: an open-source

File: blog/2022-01-20-super-fast-react-localizaton-with-i18next-and-tolgee.md [hunk 6]
```diff
@@ -0,0 +1,340 @@
...
+```
+
+To test it works, you can comment out the apiKey param. App should be translated without fetching the data from Tolgee
+API.
+
+Congrats! 🎉 Now you are able to speed up your localization process!
+
+## TL;DR
+
+- Tolgee is open-source tool, simplifying localization process and removing unnecessary tasks
```
Comment (marketachalupnikova) [inline blog/2022-01-20-super-fast-react-localizaton-with-i18next-and-tolgee.md:333]: an open-source

File: blog/2022-01-20-super-fast-react-localizaton-with-i18next-and-tolgee.md (hunk 5 above)
Comment (marketachalupnikova) [reply blog/2022-01-20-super-fast-react-localizaton-with-i18next-and-tolgee.md:28]: a simple way

File: blog/2022-01-20-super-fast-react-localizaton-with-i18next-and-tolgee.md [hunk 7]
```diff
@@ -0,0 +1,340 @@
...
+- It enables you or your colleagues to modify translated string in the context of developed web application
+- Tolgee is also localization platform, where you can manage all your translations
```
Comment (marketachalupnikova) [inline blog/2022-01-20-super-fast-react-localizaton-with-i18next-and-tolgee.md:335]: a localization platform

**Pull request merged**